packages/mux-active-viewer-count/src/index.ts

```diff
@@ -0,0 +1,225 @@
+import { globalThis } from 'shared-polyfills';
+
+export const subscribeViewerCount = (
+  token: string,
+  pollInterval: number,
+  callback: (views: number) => void,
+  errorCb: (errorMsg: string) => void
+) => {
+  const url = `https://stats.mux.com/counts?token=${token}`;
+  const controller = new AbortController();
+  const signal = controller.signal;
+  let timeoutId: number | undefined;
+  let aborted = false;
+  const fetchViewerCountPoll: () => Promise<any> = async () => {
```

Comment:
nit (non-blocking): if `await` is not used in the body of the function there is no need for `async` here.

Reply:
Good call. This was an artifact from before yielding the `unsubscribe` function. I agree, it can/should be removed (even if it doesn't have any real impact, the definition is "lying").

Reply:
Fixed!